controllers/drpolicy_controller.go

```diff
@@ -90,6 +97,67 @@ func (r *DRPolicyReconciler) Reconcile(ctx context.Context, req ctrl.Request) (c
 	return ctrl.Result{}, nil
 }
 
+func validate(ctx context.Context, drpolicy *ramen.DRPolicy, apiReader client.Reader,
+	client client.Client, objectStoreGetter ObjectStoreGetter, log logr.Logger,
+) error {
+	var (
+		conditionSetTrue  func(reason, message string) error
+		conditionSetFalse func(reason string, err error) error
+	)
+
+	if condition := util.DrpolicyValidatedConditionGet(drpolicy); condition != nil {
+		if condition.Status == metav1.ConditionTrue {
+			log.Info(`valid -> valid`)
```

Comment:
Curious; why do you use backticks instead of quotes? Backticks are for handling special characters.  https://golang.org/ref/spec#String_literals
To keep the code common, I suggest using "" unless necessary.

Reply:
I'm willing to adopt this style for the future.  Since this pull request has already been merged I opened https://github.com/RamenDR/ramen/issues/317 to track changing the code merged this pull request.